todo_app/templates/index.html

```diff
@@ -9,14 +9,22 @@ <h1 class="display-4">To-Do App</h1>
 
 <div class="row">
   <div class="col-md-12">
-    <h2>Items</h2>
+    <h2>To Do Items</h2>
     <ul class="list-group mb-4">
-      {% for item in items %}
-      <li>{{item.status}} - {{item.name}}
-        {% if item.status == 'To Do'%}
-          <a href="{{ url_for('complete_item_route',id=item.id) }}">Mark as done</a>
-        {% endif %}
-      </li>
+      {% for item in view_model.to_do_items %}
+      <li>{{item.status}} - {{item.name}} <a href="{{ url_for('complete_item_route',id=item.id) }}">Mark as done</a></li>
```

Comment:
Displaying item.status is a little redundant when this whole list is for "To Do Items" 😛